draft-irtf-cfrg-vdaf.md

```diff
@@ -1817,86 +1819,87 @@ The parameters provided by a concrete FLP are listed in {{flp-param}}.
+* `Flp.prove(meas: Vec[Field], prove_rand: Vec[Field], joint_rand: Vec[Field])
   -> Vec[Field]` is the deterministic proof-generation algorithm run by the
-  prover. Its inputs are the encoded input, the "prover randomness"
-  `prove_rand`, and the "joint randomness" `joint_rand`. The prover randomness is
-  used only by the prover, but the joint randomness is shared by both the prover
-  and verifier.
+  prover. Its inputs are the encoded measurement, the "prover randomness"
+  `prove_rand`, and the "joint randomness" `joint_rand`. The prover randomness
+  is used only by the prover, but the joint randomness is shared by both the
+  prover and verifier.
 
-* `Flp.query(input: Vec[Field], proof: Vec[Field], query_rand: Vec[Field],
+* `Flp.query(meas: Vec[Field], proof: Vec[Field], query_rand: Vec[Field],
   joint_rand: Vec[Field], num_shares: Unsigned) -> Vec[Field]` is the
   query-generation algorithm run by the verifier. This is used to "query" the
-  input and proof. The result of the query (i.e., the output of this function)
-  is called the "verifier message". In addition to the input and proof, this
-  algorithm takes as input the query randomness `query_rand` and the joint
-  randomness `joint_rand`. The former is used only by the verifier. `num_shares`
-  specifies how many input and proof shares were generated.
+  measurement and proof. The result of the query (i.e., the output of this
+  function) is called the "verifier message". In addition to the measurement
+  and proof, this algorithm takes as input the query randomness `query_rand`
+  and the joint randomness `joint_rand`. The former is used only by the
+  verifier. `num_shares` specifies how many shares were generated.
 
 * `Flp.decide(verifier: Vec[Field]) -> Bool` is the deterministic decision
   algorithm run by the verifier. It takes as input the verifier message and
-  outputs a boolean indicating if the input from which it was generated is
-  valid.
+  outputs a boolean indicating if the measurement from which it was generated
+  is valid.
 
 Our application requires that the FLP is "fully linear" in the sense defined in
 {{BBCGGI19}}. As a practical matter, what this property implies is that, when
-run on a share of the input and proof, the query-generation algorithm outputs a
-share of the verifier message. Furthermore, the privacy property of the FLP
-system ensures that the verifier message reveals nothing about the input's
-validity. Therefore, to decide if an input is valid, the Aggregators will run
-the query-generation algorithm locally, exchange verifier shares, combine them
-to recover the verifier message, and run the decision algorithm.
+run on a share of the measurement and proof, the query-generation algorithm
+outputs a share of the verifier message. Furthermore, the privacy property of
+the FLP system ensures that the verifier message reveals nothing about the
+measurement's validity. Therefore, to decide if a measurement is valid, the
```

Comment:
Unrelated to this PR: this is backwards, and contradictory to the definition of `Flp.decide()`.
```suggestion
the FLP system ensures that the verifier message reveals nothing about the measurement
other than whether it is valid. Therefore, to decide if a measurement is valid, the
```